internal/nginx/templates/nginx-plus.tmpl

```diff
@@ -96,10 +96,10 @@ http {
 
         location  = /dashboard.html {
         }
-
-        allow 127.0.0.1;
+{{ range $value := .NginxStatusAllowCIDRs }}{{ if ne $value "" }}
+        allow {{$value}};{{ end }}
         deny all;
```

Comment:
sorry, haven't caught it during the first review. Could you move the `deny all` just below the `range` loop.  
```
{{range ... }}
{{end}}
deny all;
```